packages/app-project/src/components/ProjectHeader/hooks/useStores.js

```diff
@@ -18,6 +18,14 @@ function storeMapper(store) {
     }
   } = store
 
+  const availableLocales = languages?.reduce((acc, current) => {
```

Comment:
This maps one array to another, so `Array.prototype.map` would be a better fit than `Array.prototype.reduce`